mk/db.mk

```
@@ -49,20 +48,20 @@ db-migrate-seed: $(GO_OUTPUT)/dbmigrate ## Run dbmigrate seed
 
 .PHONY: db-down
 db-down: ## Stop postgres database
-	! $(DOCKER) container inspect postgres &> /dev/null || $(DOCKER) container stop postgres
+	! $(DOCKER) container inspect $(DATABASE_CONTAINER_NAME) &> /dev/null || $(DOCKER) container stop $(DATABASE_CONTAINER_NAME)
 
 .PHONY: db-clean
 db-clean: db-down ## Clean database volume
-	! $(DOCKER) volume inspect postgres &> /dev/null || $(DOCKER) volume rm postgres
+	! $(DOCKER) volume inspect $(DATABASE_CONTAINER_NAME) &> /dev/null || $(DOCKER) volume rm $(DATABASE_CONTAINER_NAME)
 
 .PHONY: db-cli-connect
 db-cli-connect: ## Open a postgres cli in the container (it requires db-up)
-	! $(DOCKER) container inspect postgres &> /dev/null || $(DOCKER) container exec -it postgres psql "sslmode=disable dbname=$(DATABASE_NAME) user=$(DATABASE_USER) host=$(DATABASE_HOST) port=$(DATABASE_PORT) password=$(DATABASE_PASSWORD)"
+	! $(DOCKER) container inspect $(DATABASE_CONTAINER_NAME) &> /dev/null || $(DOCKER) container exec -it $(DATABASE_CONTAINER_NAME) psql "sslmode=disable dbname=$(DATABASE_NAME) user=$(DATABASE_USER) host=$(DATABASE_HOST) port=$(DATABASE_INTERNAL_PORT) password=$(DATABASE_PASSWORD)"
 
 .PHONY: db-dump-table
 db-dump-table:
-	! $(DOCKER) container inspect postgres &> /dev/null || $(DOCKER) container exec -it postgres pg_dump --table "$(DATABASE_TABLE)" --schema-only --dbname=$(DATABASE_NAME) --host=$(DATABASE_HOST) --port=$(DATABASE_PORT) --username=$(DATABASE_USER)
+	! $(DOCKER) container inspect $(DATABASE_CONTAINER_NAME) &> /dev/null || $(DOCKER) container exec -it $(DATABASE_CONTAINER_NAME) pg_dump --table "$(DATABASE_TABLE)" --schema-only --dbname=$(DATABASE_NAME) --host=$(DATABASE_HOST) --port=$(DATABASE_INTERNAL_PORT) --username=$(DATABASE_USER)
 
 .PHONY: db-shell
 db-shell:
-	! $(DOCKER) container inspect postgres &> /dev/null || $(DOCKER) container exec -it postgres bash
+	! $(DOCKER) container inspect $(DATABASE_CONTAINER_NAME) &> /dev/null || $(DOCKER) container exec -it $(_CONTAINER_NAME) bash
```

```suggestion
	! $(DOCKER) container inspect $(DATABASE_CONTAINER_NAME) &> /dev/null || $(DOCKER) container exec -it $(DATABASE_CONTAINER_NAME) bash
```
Just fix this one thing and it's good to go